nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/PublishKafkaRecord_2_6.java

```diff
@@ -312,6 +331,8 @@ public class PublishKafkaRecord_2_6 extends AbstractProcessor implements Verifia
         properties.add(KafkaProcessorUtils.TOKEN_AUTH);
         properties.add(KafkaProcessorUtils.SSL_CONTEXT_SERVICE);
         properties.add(MESSAGE_KEY_FIELD);
+        properties.add(PUBLISH_STRATEGY);
```

Comment:
Is probably best to add Publish Strategy before Message Key Field, since Message Key Field depends on Publish Strategy. Otherwise, we have a situation where changing the value of Publish Strategy introduce a new property above it, which is kind of an odd user experience.